avm/utilities/pipelines/e2eValidation/regionSelector/Get-AzAvailableResourceLocation.ps1

```diff
@@ -85,13 +96,20 @@ function Get-AzAvailableResourceLocation {
     } |  Select-Object -ExpandProperty Location
     Write-Verbose "Available Locations: $($locations | ConvertTo-Json)"
 
-
-    $index = Get-Random -Maximum ($locations.Count)
-    Write-Verbose "Generated random index [$index]"
-
-    $location = $locations[$index]
-
+    $filteredAllowedLocations = @()
```

Comment:
```suggestion
```

It's overritten right after. Should be redundant

Reply:
removed